### PR TITLE
Move to jhipster v7

### PR DESCRIPTION
**Pull request by avdev4j**
fix https://github.com/jhipster/generator-jhipster/issues/14327

### CODE REVIEW

File: generators/app/templates/package.json.ejs [hunk 1]
```diff
@@ -5,53 +5,42 @@
   "keywords": [
     "yeoman-generator",
     "jhipster-module",
-    "jhipster-6"
+    "jhipster-7"
   ],
   "homepage": "https://github.com/<%= githubName %>/generator-jhipster-<%= moduleName %>",
+  "bugs": {
+    "url": "https://github.com/avdev4j/generator-jhipster-helloworld/issues"
```
Comment (qmonmert) [inline generators/app/templates/package.json.ejs:12]: `https://github.com/avdev4j/generator-jhipster-helloworld/issues` 🤔

Comment (avdev4j) [reply generators/app/templates/package.json.ejs:12]: Oupsi

**Review by mshima — COMMENTED** [review]
There are some more things to take into account:

Are we recommending `jhipster run` or `yo`?
modules will require `yo@next-4` due to jhipster switching to yeoman-generator@5.
If we are recommending `jhipster run`, we should update ci and add arguments and options, `jhipster run` doesn't allows unknown options.

File: generators/app/templates/package.json.ejs [hunk 2]
```diff
@@ -65,14 +54,26 @@
...
-  <%_ } else if (license === 'gpl') { _%>
-  "license": "GPL-3.0",
-  <%_ } else if (license === 'mit') { _%>
-  "license": "MIT",
-  <%_ } _%>
-  "bugs": {
-    "url": "https://github.com/<%= githubName %>/generator-jhipster-<%= moduleName %>/issues"
+  "dependencies": {
+    "chalk": "4.1.0",
+    "generator-jhipster": ">=7.0.0-beta.1",
```
Comment (fdelbrayelle) [inline generators/app/templates/package.json.ejs:59]: Shouldn't it be this instead? :thinking: 

```suggestion
    "generator-jhipster": ">=7.0.0",
```

**Pull request merged**